frontend/src/app/modules/games/games-saga.js

```diff
@@ -11,11 +13,21 @@ export function* getGamesSaga() {
   }
 }
 
-export function* selectGameSaga(action) {
-  yield put(GamesActions.Creators.selectGame(action.games[0]))
+export function* selectGameSaga(game) {
+  yield put(GamesActions.Creators.selectGame(game))
+}
+
+export function* selectGameOnGamesLoadedSaga(action) {
+  yield call(selectGameSaga, action.games[0])
```

Comment:
what is that games[0] referring to? 🤔

Reply:
to foosball :) as right now we "can't" have more games, I just take the first one of one existing.

Reply:
it's temporary as long as we support only one game on UI

Reply:
if it's temporary I'd either make a comment here, or rename the saga to something like selectFoosball - so the name says what it does

Reply:
Fixed in https://github.com/salsita/foosball-rating/pull/125/commits/87707178781b189c0157f2314e3bb531c773c7c0